app/uk/gov/hmrc/customs/emailfrontend/views/email_verified_or_changed.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.customs.emailfrontend.views.html.Layout
+@import uk.gov.hmrc.customs.emailfrontend.views.html.components
+@import uk.gov.hmrc.customs.emailfrontend.viewmodels.EmailVerifiedOrChangedViewModel
+
+@this(
+  layout: Layout,
+  h1_xl: components.h1_xl,
+  link: components.link,
+  button: components.button,
+  verifiedPanel: components.emailVerifiedPanel,
+  changedPanel: components.emailPanel,
+  p: components.p
+)
+
+@(viewModel: EmailVerifiedOrChangedViewModel)(implicit request: Request[_], messages: Messages)
+
+@layout(Some(messages(viewModel.titleKey)), welshToggle = false) {
```

Comment:
Should we disable the `welshToggle` ?

Reply:
This was how it was before, so honestly I don't know. This matches what's displayed in the ticket anyway.